packages/shared/tailwind/colors.ts

```diff
@@ -93,6 +93,7 @@ const colors = {
     90: '#D9113A',
   },
   cabbage: {
+    0: '#E669FB00',
```

Comment:
We probably have this already no? it's just a gradient version.
We use the color-mix for that normally.